std/experimental/allocator/package.d

```diff
@@ -2844,7 +2857,7 @@ private struct InternalPointersTree(Allocator)
     /// Ditto
     bool deallocate(void[] b)
     {
-        if (!b.ptr) return;
+        if (!b.ptr) return true;
```

Comment:
how in the world did this pass until now?

Reply:
The only reasonable explanation is that it was never instantiated.

Reply:
This code doesn't get hit because of the `__EOF__` [here](https://github.com/edi33416/phobos/blob/d621a465202844548338c126b2ec362a18f0b48f/std/experimental/allocator/package.d#L2545).